benchexec/tablegenerator/react-table/src/components/Tooltip.js

```diff
@@ -0,0 +1,33 @@
+// This file is part of BenchExec, a framework for reliable benchmarking:
+// https://github.com/sosy-lab/benchexec
+//
+// SPDX-FileCopyrightText: 2019-2020 Dirk Beyer <https://www.sosy-lab.org>
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faInfoCircle } from "@fortawesome/free-solid-svg-icons";
+
+const IconWithTooltip = ({ message }) => {
+  return (
+    <div
+      className="infoTooltipContainer"
+      onMouseEnter={(e) => {
+        const tooltip = e.currentTarget.querySelector(".infoTooltip");
+        tooltip.style.visibility = "visible";
```

Comment:
I would prefer to have less CSS in the JavaScript code. I think usually we handle visibility of DOM elements via CSS classes, can we do that here as well?